src/app/shared/case-editor/case-edit-submit.component.ts

```diff
@@ -45,6 +45,8 @@ export class CaseEditSubmitComponent implements OnInit {
 
   ngOnInit(): void {
     this.eventTrigger = this.caseEdit.eventTrigger;
+    this.triggerText = this.eventTrigger.end_button_label ? this.eventTrigger.end_button_label
```

Comment:
2 things: 
1) I think (not 100% sure) you can shorten this with:
this.triggerText = this.eventTrigger.end_button_label || CallbackErrorsComponent.TRIGGER_TEXT_SUBMIT;

2) I know it's not your code but I wonder why the trigger button default text is inside the CallbackErrorsComponent. Seems to me in the wrong place. Perhaps we can move it inside this class?